modules/oe_translation_epoetry/src/TranslationRequestEpoetry.php

```diff
@@ -183,4 +187,72 @@ public function setEpoetryRequestStatus(string $status): TranslationRequestRemot
     return $this;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getEpoetryRequestStatusDescription(string $status): TranslatableMarkup {
```

Comment:
Some of the statuses depend on the entity to determine the status message. In the tests we dont have access to this entity as we are asserting the message that is printed on the page. So it's quite difficult to avoid this duplication if we want to do the assertions